### PR TITLE
Add support for zenscript

### LINKED ISSUE #2417: [Icon Request] Support for ZenScript
- [x] I'm sure this issue is not a _duplicate_.
- [x] I'm submitting an icon request:
  - Type: `extension`
  - Icon Name: `ZenScript`
  - Sample original Icon: logo
  - Extensions: `zs`

- More info: ZenScript is language for CraftTweaker - most popular Minecraft modpack making tool [(CraftTweaker Documentation)](https://docs.blamejared.com/1.12/en/) [(Zenscript Github)](https://github.com/CraftTweaker/ZenScript)

### PR DESCRIPTION
**Pull request by KingDarBoja**
_**Fixes #2417**_

**Changes proposed:**

- [x] Add
- [ ] Delete
- [ ] Fix
- [ ] Prepare

### CODE REVIEW

File: src/iconsManifest/supportedExtensions.ts [hunk 1]
```diff
@@ -4014,6 +4014,12 @@ export const extensions: IFileCollection = {
       filename: true,
       format: FileFormat.svg,
     },
+    {
+      icon: 'zenscript',
+      extensions: ['.zsrc.json'],
```
Comment (JimiC) [inline src/iconsManifest/supportedExtensions.ts]: This is a filename.

**Comment by JimiC** [comment]
@KingDarBoja Have you checked that this is indeed working with the highlight extension and the added filename?

**Comment by JimiC** [comment]
Also, where does support for `.zsrc.json` comes from?

**Comment by JimiC** [comment]
Furthermore, this is a `png` file. It's not acceptable.

**Comment by KingDarBoja** [comment]
Wasn't able to create a svg from the official icon as it has too many details on it.

@JimiC the `.zsrc.json` comes from [zenscript vscode extension](https://github.com/Yesterday17/ZenScript/blob/master/package.json#L142).

**Comment by JimiC** [comment]
1. Then we will have to close this PR until an `svg` is available.
2. the filename is `.zsrc` not `.zsrc.json`

**Comment by JimiC** [comment]
Closing in search of a better icon.

**Pull request closed without merging**